### ISSUE TITLE
Boat won't show when assembled in 1.16.5

### ISSUE THREAD

**Issue by TheThe-Breadman** [ValkyrienSkies/Eureka#203]
Could you see if you guys could fix it? I wanna use the mod in 1.16.5

**Comment by Woodynson** [comment]
i have the same problem does anyone know why is it doing this ? or how to fix it ? every time i click assemble the blocks just vanish

**Comment by DragonSeason** [comment]
My friend and me have the exact same problem!
It was kind of weird. First only I didn't see the assembled boat, but after a while and restarting the server several times, he couldn't see the boat either.
When only he could see the boat, on my side mobs were affected by the collision of the assembled boat and I could temporarily ride along with a vanilla boat if placed inside the invisible ship. (I was otherwise not affected by the collision and just fell through to the ocean) But just temporarily, as the vanilla boat sank into the ocean after traveling a few blocks.

It would be awesome if this bug could be fixed somehow!

**Comment by Minepro306** [comment]
same with me! it happend today in 1.19.2 and i dont know how to fix it except mine was invisible

**Comment by NullHarp** [comment]
do you have optifine?

**Comment by MaskedSquidd** [comment]
I tested it multiple times it is not compatible with optifine, I took it out and it showed up however it does work with Rubidium and Oculus, (Forge Iris Ports) Also use them together) which also allow shaders and recommend this to fix the issue

**Comment by DragonSeason** [comment]
No, neither of us used optifine or any mod that would have similar functionality. The problem still persists and I really wish this would be fixed.

**Comment by walksanatora** [comment]
how about giving the entire mod list :>

**Comment by DragonSeason** [comment]
> how about giving the entire mod list :>

It happens with just the essential mods. No additional mods were added.
